doc/source/extending.rst

```diff
@@ -160,9 +160,18 @@ your ``MyExtensionArray`` class, as follows:
     MyExtensionArray._add_arithmetic_ops()
     MyExtensionArray._add_comparison_ops()
 
-Note that since ``pandas`` automatically calls the underlying operator on each
-element one-by-one, this might not be as performant as implementing your own
-version of the associated operators directly on the ``ExtensionArray``.
+
+.. note::
+
+   Since ``pandas`` automatically calls the underlying operator on each
+   element one-by-one, this might not be as performant as implementing your own
+   version of the associated operators directly on the ``ExtensionArray``.
+
+This implementation will try to reconstruct a new ``ExtensionArray`` with the
+result of the element-wise operation. Whether or not that succeeds depends on
+whether the operation returns a result that's valid for the ``ExtensionArray``.
+If an ``ExtensionArray`` cannot be reconstructed, a list containing the scalars
+returned instead.
```

Comment:
I find it a bit strange we return a list and not an array ? 
(but that's maybe off topic for this PR)

Reply:
Yes, I found that strange as well. An array would be better to return.

This is new in 0.24 right? If so, I'll just make the change here.

Reply:
I have another PR touching about the same place right now, so I'm going to hold off on changing that till later.